exam-01/section-1.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Section 2: `15/15`
Section 3: `10/20`

It's good that you have fixed up all the debugging issues. However, when applying bootstrap styling, you copied without checking so at the same time you have removed all the required `data-component` that are necessary to have your page running correctly.